scheduler/src/cook/mesos/rebalancer.clj

```diff
@@ -155,7 +155,27 @@
   [dru]
   (* dru metrics-dru-scale))
 
-(defn compute-pending-job-dru
+(defn compute-pending-gpu-job-dru
+  [{:keys [task->scored-task user->sorted-running-task-ents user->dru-divisors] :as state}
+   pending-job-ent]
+  (let [user (:job/user pending-job-ent)
+        {gpu-req :gpus} (util/job-ent->resources pending-job-ent)
+        gpu-divisor (get-in user->dru-divisors [user :gpus])
+        pending-task-ent (util/create-task-ent pending-job-ent)
+        nearest-task-ent (some-> user->sorted-running-task-ents
```

Comment:
Correct. It's defined over the comparator, which is defined by `same-user-task-comparator` in `util.clj`, which is sorted first by highest priority, then earliest submission time, then tiebreakers.